src/store/getters.js

```diff
@@ -26,9 +27,12 @@ export default {
   accounts({ accounts: { list } }, getters) {
     if (!getters.wallet) return [];
     return list
-      .map(({ idx, type, ...acc }) => ({
+      .map(({
+        idx, type, protocol, ...acc
+      }) => ({
         idx,
         type,
+        protocol: protocol || PROTOCOL_AETERNITY,
```

Comment:
The existing accounts do not have any protocol type and the existing accounts are always Aeternity accounts.